cmake/CppInterOp/CppInterOpConfigVersion.cmake.in

```diff
@@ -1,4 +1,4 @@
-set(PACKAGE_VERSION "@CPPINTEROP_VERSION@")
+set(PACKAGE_VERSION "@CPPINTEROP_VERSION_MAJOR@.@CPPINTEROP_VERSION_MINOR@.@CPPINTEROP_VERSION_PATCH@")
```

Comment:
We need to signal if that came from a dev version or not somehow.

Reply:
I could have a variable contain the last thing in the version list. If it is the word dev then we could output a message to the screen telling the user that they are working with a dev version.

Reply:
On a second thought let's ignore this for now. Maybe we can keep track of this problem in a separate issue.

Reply:
We have managed to retain the ;dev part in package version in the end.